nycnoise/settings/base.py

```diff
@@ -195,3 +197,10 @@ def custom_show_pyinstrument(request):
     MIDDLEWARE += [
         "pyinstrument.middleware.ProfilerMiddleware",
     ]
+
+# Django-meta configuration
+META_USE_OG_PROPERTIES = True
+META_USE_TWITTER_PROPERTIES = True
+META_USE_TITLE_TAG = True
+META_USE_SITES = True
```

Comment:
this forces it to call db to fetch the site name, we can consider hardcoding it instead